lnbits/wallets/boltz.py

```diff
@@ -95,8 +95,10 @@ async def create_invoice(
             wallet_id=self.wallet_id,
             accept_zero_conf=True,
             external_pay=True,
-            description=memo,
         )
+        if memo is not None:
+            # boltz rejects nbsp char (produced by JS Intl.NumberFormat api)
+            request.description = memo.replace(" ", " ")  # noqa: RUF001
```

Comment:
IMO this is better: 

```suggestion
            request.description = memo.replace("\xa0", " ")
```

Reply:
I agree, its much clearer.
@jackstar12 can you please test and commit if all good

Reply:
ill test real quick

Reply:
works